packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/__tests__/SimpleDocument.test.tsx

```diff
@@ -45,4 +47,41 @@ describe('SimpleDocument', () => {
       { exact: false }
     );
   });
+
+  it('renders a document with a different body field', () => {
+    const documentText = 'This is the text of the document.';
+    const customDoc = {
+      body_field: documentText
+    };
+    const context: Partial<SearchContextIFC> = {
+      componentSettings: {
+        fields_shown: {
+          body: {
+            field: 'body_field',
+            use_passage: true
```

Comment:
i guess this code path doesn't care what `use_passage` is set to (in tooling, `body.field` is ignored in the search results if `use_passage` is `true` -> does the `DocumentPreview` still use the field from `document_passage[0].field` when highlighting?)

Reply:
For passage highlighting, we'll pull the field from the `passage` data (or, now, from the `body.field`). Is that what you're asking?

Reply:
Right, in saying when use_passage is `true` you should always pull from the field specified in the passage data, even if body field is specified

Reply:
Ah, right, I get you. Not only `body.use_passage`, but if we have a `passage` passed in, use that first; **then**, look in `body.field || text`.

Current code has issue that if `passage.field` is "text", we may instead be using `body.field`.

Reply:
@ColbyJohnIBM I added a test which tests if passage field takes precedence over body field. That failed, so I moved the code around to handle this issue, too. Take a look: https://github.com/watson-developer-cloud/discovery-components/pull/55/commits/58aa82978a8dca4a7bf4708a4b3bb51787671f96